### PR TITLE
12 fix: DHIS2-9140 adds correct texts and translations

### PR DESCRIPTION
**Pull request by paschalidi**
Hey @JoakimSM this is another small ticket about text and i18n https://jira.dhis2.org/browse/DHIS2-9140

Have a look and let me know :)

### CODE REVIEW

File: src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js [hunk 1]
```diff
@@ -150,7 +152,7 @@ const Index = ({
                             data-test="dhis2-capture-general-purpose-error-mesage"
                             className={classes.generalPurposeErrorMessage}
                         >
-                            {generalPurposeErrorMessage}
+                            {i18n.t(generalPurposeErrorMessage)}
```
Comment (JoakimSM) [inline src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js:155]: This won't work, you need to use i18n.t where the actual text is defined.

**Review by JoakimSM — COMMENTED** [review]
One comment for you @paschalidi then this should be ok 👍

File: src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js (hunk 1 above)
Comment (paschalidi) [reply src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js]: Dont see the reason to have this stored in redux. I removed it from there.

**Comment by paschalidi** [comment]
@JoakimSM perhaps you can go on another round on this one

**Review by JoakimSM — APPROVED** [review]
LGTM 👍

**Pull request merged**